yucca/functional/evaluation/evaluate_folder.py

```diff
@@ -13,195 +13,228 @@
 from yucca.functional.evaluation.metrics import auroc
 
 
-def evaluate_multilabel_folder_segm(
+def evaluate_folder_segm(
     labels: dict,
-    metrics,
-    subjects,
-    folder_with_predictions,
-    folder_with_ground_truth,
+    metrics: dict,
+    subjects: list,
+    folder_with_predictions: str,
+    folder_with_ground_truth: str,
     as_binary: Optional[bool] = False,
     obj_metrics: Optional[bool] = False,
-    regions: Optional[list] = None,
     surface_metrics: Optional[bool] = False,
-    surface_tol=1,
+    surface_tol: int = 1,
+    regions: Optional[dict] = None,
+    multilabel: bool = False,
```

Comment:
Minor pet peeve: can we have args in alphabetical? so like required args first in alphabetical, then optionals in alphabetical. 
(I know I might've not done it myself, but now I cannot unsee it)